Containerfile.externaldns

```diff
@@ -28,8 +28,8 @@ FROM registry.access.redhat.com/ubi9/ubi:9.6-1752625787
 LABEL maintainer="Red Hat, Inc."
 LABEL com.redhat.component="external-dns-container"
 LABEL name="external-dns"
-LABEL version="1.3.0"
-LABEL commit="76d92ad82b22c92c191a8c0145d3712e4012d987"
+LABEL version="1.3.1"
+LABEL commit=${FULL_COMMIT}
```

Comment:
You pass `FULL_COMMIT` as a building arg however it's not set as `ARG` for this Containerfile.